### DISCUSSION TITLE
Convert HTML in String

### DISCUSSION THREAD

**Discussion by atacan** [vapor-community/HTMLKit#77] [Ideas]
### The use case:
The developer works with a CSS framework like Bootstrap and copy-paste various small components from the internet and customises them.
### Solutions:
* Make a playground with HTMLKit converter and paste the HTML code into a string. Copy the resulting swift code into the clipboard. Paste the code into your Vapor project.
* Make a small Mac app with text field to paste the HTML code. One click on a button converts and puts the result into a clipboard, while showing it to the user on the UI. Paste the code into your Vapor project.
### To Do:
Currently the converter accepts only a file path for the HTML code. The result is either printed or written into a file.
* Add a function that uses `XMLDocument(xmlString:` and gets a string as an argument
* Add an option to return the result as string.

**Comment by mattesmohr** [comment]
The first version of the converter already had a string to string option. I can't remember why I didn't implement it in the current version. I guess my thought was biased. 

With your use case in mind, I think it's okay to bring the string to string back.
